app/gunicorn.conf.py

```diff
@@ -0,0 +1,8 @@
+import os
```

Comment:
can we add a file level docstring about what this file is for and maybe link to https://docs.gunicorn.org/en/stable/configure.html

Reply:
Thanks for holding us to a high level of documentation! You really set the bar high here.